src/helpers/validations/userValidator.js

```diff
@@ -0,0 +1,60 @@
+import { check, validationResult } from 'express-validator/check';
```

Comment:
#### Critical
You were told not to use `express-validator`. What's this doing here?